nvflare/fuel/utils/function_utils.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import importlib
+import inspect
+from typing import Callable, Tuple
+
+
+def find_task_fn(task_fn_path: str) -> Callable:
+    """
+    Find and return a callable task function based on its module path.
+
+    Args:
+        task_fn_path (str): The path to the task function in the format "module_path.function_name".
+
+    Returns:
+        Callable: The callable task function.
+    """
+    # Split the text by the last dot
+    tokens = task_fn_path.rsplit(".", 1)
+    module_name = tokens[0]
+    fn_name = tokens[1] if len(tokens) > 1 else ""
+    module = importlib.import_module(module_name)
+    fn = getattr(module, fn_name)
+    return fn
+
+
+def require_arguments(func: Callable) -> Tuple[bool, int, int]:
+    """
+    Check if a function requires arguments and provide information about its signature.
```

Comment:
All string and code looks good.

If we want to be 100% compatible to google style: https://google.github.io/styleguide/pyguide.html#381-docstrings

The summary begins after `"""`, so `"""Checks if a function requires arguments and provide information about its signature.`

But we have other places that were not followed strictly, so we can fix this later as well.